dialog/file.go

```diff
@@ -77,17 +82,29 @@ func (f *fileDialog) makeUI() fyne.CanvasObject {
 		if f.file.save {
 			callback := f.file.callback.(func(fyne.URIWriteCloser, error))
 			name := f.fileName.(*widget.Entry).Text
-			path := filepath.Join(f.dir, name)
+			path, _ := storage.Child(f.dir, name)
+
+			// On windows replace '\\' with '/'
+			if runtime.GOOS == "windows" {
+				pathString := path.String()
```

Comment:
path is now a URI so this should not be needed

Reply:
I changed it back